### PR TITLE
user approval

### PR DESCRIPTION
**Pull request by cezio**
user approval #255, ported from https://github.com/GeoNode/geonode-user-accounts

### CODE REVIEW

**Comment by afabiani** [comment]
@cezio @brosner how is the state of those PRs? How long to merge them?

File: account/views.py [hunk 1]
```diff
@@ -351,6 +357,22 @@ def closed(self):
         }
         return self.response_class(**response_kwargs)
 
+    def account_approval_required_response(self):
+        if self.request.is_ajax():
+            template_name = self.template_name_admin_approval_ajax
```
Comment (blag) [inline account/views.py:362]: This is misspelled - it should be `template_name_admin_approval_sent_ajax`.

**Review by blag — CHANGES_REQUESTED** [review]
Can you also add an empty `account/ajax/admin_approval_sent.html` file?

And one typo fix. Otherwise this looks good.

**Comment by blag** [comment]
Oh, and it would be nice to document the two templates that you've introduced:
* `account/admin_approval_sent.html`
* `account/ajax/admin_approval_sent.html`

**Comment by blag** [comment]
Oh, and it would be nice to document the two templates that you've introduced:
* `account/admin_approval_sent.html`
* `account/ajax/admin_approval_sent.html`

File: account/views.py (hunk 1 above)
Comment (cezio) [reply account/views.py]: thanks for the review! should be fixed now.

Comment (blag) [reply account/views.py]: Cool, thanks! Hopefully I can be a maintainer for this repository soon.

One thing I'd eventually like to do is write some tests for this project. If you have the time, can you write some tests for this feature?

**Review by afabiani — APPROVED** [review]

File: account/views.py (hunk 1 above)
Comment (cezio) [reply account/views.py]: Tests already included: https://github.com/pinax/django-user-accounts/pull/256/files#diff-15a330fd5d8811681fa96af031169262fef751ecbe6bf7b8cfa39438e6a1d47d

**Review by blag — APPROVED** [review]

**Pull request merged**